packages/angular/build/src/builders/unit-test/schema.json

```diff
@@ -44,7 +44,7 @@
     },
     "watch": {
       "type": "boolean",
-      "description": "Run build when files change."
+      "description": "Re-run tests when source files change. Defaults to `false` in TTY environments and `true` otherwise."
```

Comment:
Is the default flipped here? I assume we _do_ watch in TTY environments and _don't_ outside of them? (Ref https://nodejs.org/api/tty.html#tty)